firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.14
+
+fixed - Updated file naming for bq imports (#589)
+
+fixed - Remove double quotes on string arrays (#614)
```

Comment:
these are both updates to npm scripts that don't affect the extension itself, so it doesn't actually need a version bump or re-release. Just needs an npm script release (see "Publish an npm package" in release playbook)

Reply:
Got it! Thanks for the explanation.